js/utils.js

```diff
@@ -1511,47 +1511,58 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
   }
 
-  function setFilterValues(data) {
+  function setFilterValues(options) {
     return new Promise(function(resolve) {
-      if (!data.filterOptions.length) {
+      if (!(options.config.filterOptions || []).length) {
```

Comment:
Add `options = options || {}` at the top. There needs to be more error checking.

And check if `options.config` exists too.